plugins/modules/ipaconfig.py

```diff
@@ -482,13 +482,14 @@ def main():
             required_sid = any([netbios_name, add_sids])
             if required_sid and not enable_sid:
                 ansible_module.fail_json(
-                    "'enable-sid: yes' required for 'netbios_name' "
-                    "and 'add-sids'."
+                    msg="'enable-sid: yes' required for 'netbios_name' "
+                        "and 'add-sids'."
                 )
             if enable_sid:
                 if not has_enable_sid:
-                    ansible_module.fail_json(
-                        "This version of IPA does not support 'enable-sid'.")
+                    ansible_module.fail_json(msg=(
```

Comment:
Why are here additional brackets?

Reply:
To cope with maximum line length, without splitting the string.

Reply:
Fixed.